ofctrl/fgraphMeter.go

```diff
@@ -40,9 +44,13 @@ func (self *Meter) Type() string {
 	return "meter"
 }
 
-func (self *Meter) GetFlowInstr() openflow13.Instruction {
-	meterInstr := openflow13.NewInstrMeter(self.ID)
-	return meterInstr
+func (self *Meter) GetFlowInstr() openflow15.Instruction {
```

Comment:
If meter is not working as Instruction, I think you should remove this function.

Reply:
Done. Now the caller would create Meter as an action just like any other actions.